etna/lib/commands.rb

```diff
@@ -38,6 +40,20 @@ def set_environment(env)
   class Config
     include Etna::CommandExecutor
 
+    class Show < Etna::Command
+
+      boolean_flags << '--all'
+
+      def execute(all: false)
```

Comment:
Little utility to show the user's current config.  Figure it'd be useful for debugging.